rmgpy/molecule/parser.py

```diff
@@ -571,6 +571,7 @@ def convert_3_atom_2_bond_path(start, mol):
     electrons that the molecule should undergo.
     """
     from rmgpy.data.kinetics.family import ReactionRecipe
+    from .molecule import bond_orders
 
     def is_valid(mol):
         """Check if total bond order of oxygen atoms is smaller than 4."""
```

Comment:
For some reason, I can't comment directly on line 581, but could that line be replaced with `getBondOrdersForAtom`?